src/cli.ts

```diff
@@ -234,15 +244,15 @@ async function main(): Promise<void> {
 
     console.log('📦 Installing dependencies')
     await exec(
-        'npm',
+        'yarn',
```

Comment:
We should definitely use yarn in our repo for consistency, but should we really force extension authors to use `yarn`? I'm worried that some might not even have it installed (especially if they're not from the JS ecosystem) and then it becomes _one more thing_ they need to figure out they have to install manually in addition to just NodeJS. And then if you go to https://yarnpkg.com, it shows you the documentation for Yarn v2, but we're relying on Yarn v1 here and Yarn v2 will not work. All this could be very confusing for a user.

With this concern, the question for me becomes "what is the incremental value extension authors get out of yarn over npm?" (and does it outweigh the concern), and I don't see any significant value. They have few dependencies, so performance difference is not noticeable, and latest `npm` is almost as fast as yarn (and no longer has the annoying lockfile bugs that made us switch originally).